src/suggestion-box.ts

```diff
@@ -0,0 +1,150 @@
+import { LitElement,html,css } from "lit";
+import { customElement,property,state } from "lit/decorators.js";
+
+@customElement("aalam-sgn-box")
+export class AalamSuggestionBoxElement extends LitElement {
+    constructor() {
+        super();
+        this.input.addEventListener("keydown",this.handlearrow.bind(this));
+    }
+    static styles = css`
+    :host {
+      display: grid;
+      justify-content:center;
+      align-items:center; 
+      margin:50px;  
+    }}`;
+
+    @property()
+    countheight:number = 0;
+
+    @property()
+    result:any[] = []
+
+    @property()
+    input:any = document.getElementById('inputbox');
```

Comment:
Not safe to assign the input element like this. As I said in the earlier comment, connectedCallback() is the assured place where the element got added to the dom.